contrib/pylightning/lightning/plugin.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, stdout=None, stdin=None, autopatch=True):
         if not stdin:
             self.stdin = sys.stdin
 
-        if os.getenv('LIGHTNINGD_PLUGIN') and autopatch:
+        if autopatch:
```

Comment:
This patch is wrong, AFAICT.

Reply:
Since the `LIGHTNINGD_PLUGIN` was moved after the `io_loop` for startup plugins initialization, this would prevent some plugins to be initiated at startup. I'll amend this anyway.